test/bin/build_images.sh

```diff
@@ -424,14 +435,17 @@ while getopts "iIg:st:h" opt; do
             GROUP="$(basename "$(dirname "$(realpath "${OPTARG}")")")"
             ;;
         *)
-            echo "ERROR: Unknown option ${opt}"
-            echo
-            usage
+            usage "ERROR: Unknown option ${opt}"
             exit 1
             ;;
     esac
 done
 
+if [ -n "${LAYER}" ] && [ -n "${GROUP}" ] ; then
+    usage "The layer and group options are mutually exclusive"
```

Comment:
Do they have to be? Maybe we can undo that requirement later?

Reply:
It's more of a logical problem. If I specify a group, a layer is already implied. It's similar for the template, which is on the 3rd level. 
Maybe, we should have all the 3 of layer, group and template mutually exclusive?